pkg/account/account.go

```diff
@@ -114,7 +114,11 @@ func (p *oauthPayload) domain() string {
 
 // New returns an [Account] that can be used to fetch a [vehicle.Vehicle].
 // Optional userAgent can be passed in - otherwise it will be generated from code
-func New(oauthToken, userAgent string) (*Account, error) {
+func New(ts oauth2.TokenSource, userAgent string) (*Account, error) {
```

Comment:
Currently tokens are invalidated immediately after another refresh token get issued. I'm afraid that building this functionality is going to create downstream errors for folks who manages their tokens cross app/manually

Reply:
To be fair, this PR is from before that change.